tools/report-converter/README.md

```diff
@@ -41,8 +42,8 @@ optional arguments:
                         This directory will be used to generate CodeChecker
                         report directory files.
   -t TYPE, --type TYPE  Specify the format of the code analyzer output.
-                        Currently supported output types are: asan, tsan,
-                        ubsan, msan, clang-tidy.
+                        Currently supported output types are: ubsan, tsan,
+                        infer, asan, cppcheck, msan, clang-tidy.
```

Comment:
Would it be better to add some links to the tools in the help message? I was thinking about something like this:
```
Currently supported output types are:
ubsan
tsan
infer - Facebook infer, https://fbinfer.com/
asan
cppcheck
msan
clang-tidy
```

What do you think?

Reply:
I think it can be a good idea but I will not do it in this pull request. I created a separate issue for this: #2593.